vision/detect/detect.go

```diff
@@ -170,6 +174,7 @@ func detectText(w io.Writer, file string) error {
 	return nil
 }
 
+// [START vision_detect_document]
```

Comment:
insert a newline after this region tag. otherwise it becomes part of the godoc comment.

Reply:
@broady : Fixed (and with subsequent region tags, as well)